Source/RapyutaSimulationPlugins/Private/Core/RRSDFParser.cpp

```diff
@@ -261,7 +261,15 @@ bool FRRSDFParser::ParseModelUESpecifics(const sdf::ElementPtr& InModelElement,
         endEffectorElement = endEffectorElement->GetNextElement(SDF_ELEMENT_END_EFFECTOR);
     }
 
-    // 6- WholeBody's material
+    // 6- WholeBody's static mesh (for World model only to make use of pre-baked UE static mesh)
+    sdf::ElementPtr staticMeshElement = ueElement->FindElement(SDF_ELEMENT_LINK_STATIC_MESH);
```

Comment:
is assumption input xml has only one static mesh?
Please write those in somewhere when you write doc